content/metadata-stats/stats.textile

```diff
@@ -27,39 +27,103 @@ redirect_from:
   - /docs/realtime/versions/v0.8/statistics
 ---
 
-Ably provides statistics at one minute intervals. Statistics are aggregated by minute, hour, day and month. They are available as:
+Ably provides statistics at one minute intervals. They are aggregated by minute, hour, day and month. The most recent statistics can be delayed by up to six seconds.
 
-* "Account-level statistics":#account covering all applications in your account
-* "App-level statistics":#app for each individual application
+Statistics are available as:
 
-Account-level and app-level statistics can also be retrieved from the "Control API.":/docs/account/control-api
+* "Account statistics":#account covering all applications in your account
+* "App statistics":#app for each individual application
 
-<aside data-type='note'>
-<p>The most recent statistics are delayed by up to six seconds.</p>
-</aside>
+h2(#account). Account statistics
 
-h2(#account). Account-level statistics
+Account statistics provide information about all applications in your account.
 
-Account-level statistics are available as graphs, as tabular data, and for download from your "application dashboard":https://ably.com/accounts/any/usage_history. They are used to calculate your monthly quota and any potential "overages":https://faqs.ably.com/what-happens-if-i-exceed-my-package-connection-channel-or-message-limits.
+You can retrieve account statistics from:
 
-h2(#app). App-level statistics
+* The "Control API or REST API":#account-api
+* Your "account dashboard":#account-dashboard
 
-App-level statistics are available as graphs, as tabular data, and for download from your "application dashboard":https://ably.com/accounts/any/apps/any for each application in your account. They can also be subscribed to using the "@[meta]stats:minute@ metachannel":/docs/metadata-stats/metadata/subscribe#stats.
+h3(#account-api). Account statistics via API
+
+You can retrieve account statistics from the "Control API":/docs/account/control-api or the "REST API":/docs/api/rest-api.
+
+h4(#account-control). Account statistics via Control API
+
+You can retrieve account statistics using the "Control API":/docs/account/control-api by making a call to the @accounts@ endpoint using your @accountId@ and @accessToken@. For example, to retrieve the last two minutes of account statistics:
+
+```[curl]
+curl --location --request POST 'https://control.ably.net/v1/accounts/${ACCOUNT_ID}/stats' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer ${ACCESS_TOKEN}' \
```

Comment:
Is this and other usages of `${ACCESS_TOKEN}` meant to render an access token? It doesn't seem to be.

Reply:
No, it isn't a variable (same as `ACCOUNT_ID`) that we replace at runtime yet.